### ISSUE TITLE
Stoped working win10 update ver 2004

### ISSUE THREAD

**Issue by HellPhantom** [petrroll/PowerSwitcher#28]
Wonderful app, don't want to be without it.

After windows update to version 2004 it said it was not compatible and would not start up anymore, so I reset the app under the apps and features and it then disappeared from installed app list.

I can no longer install it from the win store, the install fails around 90%. It is almost of there is some bits off the app left somewhere on the machine that it can not replace. I Dont know where to clean or check.

Works fine on my on my other 4 machines after update to latest version 2004.

**Comment by nelson-tbone** [comment]
Hi, try to find it with this link: https://www.microsoft.com/en-us/store/p/powerplanswitcher/9nblggh556l3
Or try do find it in store with the name altogether: powerplanswitcher

I nave updated to Win10 to 2004 and here is working,

Hope I can help.

Regards.

**Comment by HellPhantom** [comment]
Thanks for responding. 

I have noticed that the nvidia drivers on one of my other pc's complained that powerplanswitcher is preventing it from upgrading its drivers and had to be stopped first.

I then went back to this problem pc which now did not have powerplan on and upgraded the nvidia drivers which could also not upgrade, so I uninstalled the Nvidia Gforce expierience completely and downloaded the latest version and installed it, it needed to reboot to uninstall because uninstall failed.

After new version was installed the install of the powerplanswitcher automatically succeeded because it was still stuck in the store as trying to install and everything was working again.

I have seen this before on all my nvidia pc's that it requests to stop the powerplan when upgrading so they must share some resources, but i would not normally relate the powerplan to video card drivers and I could not remember this because I only upgrade roughly once a year.

Thanks it is working fine again.
